src/rules/convert_require/rojo_sourcemap.rs

```diff
@@ -219,10 +272,17 @@ impl RojoSourcemap {
         ids
     }
 
-    fn find_node(&self, path: &Path) -> Option<&RojoSourcemapNode> {
-        self.root_node
-            .iter()
-            .find(|node| node.file_paths.iter().any(|file_path| file_path == path))
+    fn find_node(&self, target_path: &Path) -> Option<&RojoSourcemapNode> {
+        self.root_node.iter().find(|node| {
+            node.file_paths.iter().any(|file_path| {
+                if file_path.is_absolute() {
+                    file_path.to_path_buf()
+                        == path::absolute(target_path).expect("failed to convert")
```

Comment:
I would like to avoid calling `path::absolute` since this does not work well when darklua runs in-memory. Ideally we'd pre-compute all we need inside the initialize method.